assets/src/edit-story/app/story/effects/test/useLoadStory.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { mount } from 'enzyme';
```

Comment:
/to @swissspidy to decide if we're ok using enzyme for this. So far we've been using https://testing-library.com/ which i think has lots of the same things.

Reply:
We should definitely not start mixing testing libraries here.

Reply:
+1, updated to use @testing-library/react-hooks instead.